src/cpp/proxy/requestsession.cpp

```diff
@@ -852,7 +853,7 @@ class RequestSession::Private : public QObject
 			adata.channelPrefix = route.prefix;
 
 			acceptRequest = new AcceptRequest(acceptManager, this);
-			connect(acceptRequest, &AcceptRequest::finished, this, &Private::acceptRequest_finished);
+			finishedConnection = acceptRequest->finished.connect(boost::bind(&Private::acceptRequest_finished, this));
```

Comment:
This is a different signal/slot pair than the one for inspectRequest. I think the two connections don't occur at the same time so it could be fine to reuse `finishedConnection` for both, but it having different connection members for each might be less error prone.